### PR TITLE
Add German (Deutsch) locale

### PR DESCRIPTION
**Pull request by Branchverse**

### CODE REVIEW

**Comment by htachoires** [comment]
Hello @Branchverse, thank you for your contribution !

As I review your PR, the translation file looks good.

I found out (in game) that the file name doesn't match the locale code for deutch langage: [locale codes in minecraft](https://minecraft.fandom.com/wiki/Language)

Lastly, in the README.md, I used the DE emoji that you can found here: [deutch/german emoji](https://emojipedia.org/search?q=german)

Free to do the update, cya next time :)

File: README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 * 🇬🇧 English
 * 🇫🇷 French
+* DE German
```
Comment (htachoires) [inline README.md]: Use the DE emoji that can be found here: [deutch/german emoji](https://emojipedia.org/search?q=german)

File: README.md [hunk 2]
```diff
@@ -129,4 +130,4 @@ setting, updating or using another waypoint.
 ![commands-wp-undo](images/commands-wp-undo.png)
 
 Waypoints can be cleared by executing `/wp clear`. Be careful, this action will remove all player waypoints and cannot
-be rollback.
\ No newline at end of file
+be rollback.
```
Comment (htachoires) [inline README.md]: No need LF here

File: src/main/resources/assets/shw/lang/ger_de.json
Comment (htachoires) [inline src/main/resources/assets/shw/lang/ger_de.json:1]: I found out (in game) that the file name doesn't match the locale code for deutch langage: [locale codes in minecraft](https://minecraft.fandom.com/wiki/Language)

Comment (htachoires) [reply src/main/resources/assets/shw/lang/ger_de.json:1]: It must be _de_de.json_ to work

**Comment by Branchverse** [comment]
Perfect, that were the 2 things I didn't know where to get the information from, will fix it in time.

File: README.md (hunk 2 above)
Comment (Branchverse) [reply README.md]: That is actually done automatic can't even change it back. (Deleting the LF always gives a commit with "This commit has no content" and adds it again :/)

**Comment by htachoires** [comment]
I think you got some trouble to fix the LF x) Are you using an IDE that add them automatically ?

**Comment by htachoires** [comment]
Release available : https://www.curseforge.com/minecraft/mc-mods/set-home-waypoints/files/6174102

**Pull request merged**